src/shogun/transfer/multitask/MultitaskKernelMaskPairNormalizer.h

```diff
@@ -188,7 +188,7 @@ class CMultitaskKernelMaskPairNormalizer: public CKernelNormalizer
 	 * @param task_rhs task_id on right hand side
 	 * @return similarity between tasks
 	 */
-	float64_t get_similarity(int32_t task_lhs, int32_t task_rhs)
+	float64_t get_similarity(int32_t task_lhs, int32_t task_rhs) const noexcept
 	{
 
```

Comment:
e.g. is the noexcept a good idea here?

Reply:
The function doesn't seem to do anything that would throw an exception, but it's probably a bad decision. Should've stuck with simple getters :"D

Reply:
returning ints is fine, but the problem is that "noexept is a promise that is very hard to take back" as mentioned in the stackoverflow post. So while no exception is thrown here, maybe that changes at some point, since it is a nontrivial function

Reply:
this might be a bit too far, but still unless you get  bad allocation error I don't see what else would throw?